notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -14,16 +14,37 @@
   },
```

Comment:
<p>If we can clear this cell, it will look nicer IMO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuvs/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>